api/prisma/migrations/20210915085327_init/migration.sql

```diff
@@ -11,3 +11,15 @@ CREATE TABLE "Game" (
 
     CONSTRAINT "Game_pkey" PRIMARY KEY ("id")
 );
+
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "email" TEXT NOT NULL,
+    "hash" TEXT NOT NULL,
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "User_email_key" ON "User"("email");
```

Comment:
I think you could just add a `UNIQUE` constraint on the `email` field in the `CREATE TABLE` part  ( https://www.prisma.io/docs/guides/general-guides/database-workflows/unique-constraints-and-indexes/postgresql#2-create-a-table-with-a-single-column-unique-constraint-and-index ) and it will automatically add this index, that's what the doc says

Reply:
done automatically by Prisma, but may be customized before a launch in prod.